deployment-aws/globus_setup.md

```diff
@@ -0,0 +1,233 @@
+## Configuring Globus for use with WebMeV
+
+Note that the initial Globus endpoint and transfer node setup is a manual process. After this has been completed, configuration files can be saved, which will enable future deployments without any manual configuration. To avoid conflicts, this process needs to be repeated for each deployment you plan to create.
+
+In the end, you will need the following items saved **before** you destroy the infrastructure:
+- the deployment key, which is a JSON file typically named `deployment-key.json`
+- the node configuration, which is a JSON file created towards the end of the process.
+
+Make sure those are saved (and not added to version control!) before you destroy your temporary EC2 instance used below.
+
+### Initial setup
+
+To create the initial configuration, you will need to do the following:
+
+- Create an S3 bucket that Globus will use. Files can be read and written here by Globus. The WebMeV deployment process will also create this bucket, but here we use it as a means to test the manual setup. It will be destroyed at the end.
+    - As seen in `storage.tf`, we expect a deployment-specific bucket named `<stack>-webmev-globus` where `<stack>` is the name of the terraform workspace. *Name your bucket to follow this convention.*
+- Create an IAM user and access keys 
+    - The name does not matter, but name it so it's clearly associated with your deployment stack, so as not to accidentally delete the in the case of multiple deployments.
+    - The IAM user will be *permanent*. Otherwise, the Globus app will require manual configuration each time to update the user-associated key/secret.
```

Comment:
Ultimately, Globus requires that I give an access key/secret for some entity with read/write permissions on our bucket (`<stack>-webmev-globus`). However, at least via the console, I am only able to associate roles with either AWS services, other AWS accounts, etc. (I don't see an obvious choice for this situation). An IAM user works, but I agree it's more role-like.

Reply:
OK. It's just odd to me that Globus needs AWS keys since actual transfers are handled by the nodes directly, so a role with appropriate permissions assigned to a node instance should be sufficient.